tests/posix/common/src/clock.c

```diff
@@ -22,6 +22,9 @@ static const struct timespec ref_ts = {1514821501, NSEC_PER_SEC / 2U};
 static const clockid_t clocks[] = {
 	CLOCK_MONOTONIC,
 	CLOCK_REALTIME,
+#ifdef CONFIG_POSIX_THREAD_CPUTIME
+	CLOCK_THREAD_CPUTIME_ID
```

Comment:
I've add it.

Reply:
Please add a `settable[]` entry, and also one for `args[]` down below (a valid input).